Resources/Textures/Objects/Weapons/Melee/survival_knife.rsi/meta.json

```diff
@@ -1,26 +1,29 @@
 {
-  "version": 1,
-  "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from /tg/station at commit https://github.com/tgstation/tgstation/commit/6cdc0503d32e249125151e369edf951e93bccd1a",
-  "size": {
-    "x": 32,
-    "y": 32
-  },
-  "states": [
-    {
-      "name": "icon"
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Taken from /tg/station at commit https://github.com/tgstation/tgstation/commit/6cdc0503d32e249125151e369edf951e93bccd1a, storage sprite by RiceMar1244",
```

Comment:
Same as above. In fact, I'm pretty sure the survival knife is *actually* from Desert Rose, so you can just copy paste the text above.